sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -103,6 +103,8 @@ if (SYCL_ENABLE_XPTI_TRACING)
   set(XPTI_PROXY_SRC "${CMAKE_SOURCE_DIR}/../xpti/src/xpti_proxy.cpp")
 endif()
 
+find_package(Threads REQUIRED)
```

Comment:
It seems alternatively it can be fixed by removing Threads::Threads below (line 119).
At least it helped me locally.

Reply:
Running testing here: https://github.com/intel/llvm/pull/6060

Reply:
I thought this is what causes it to be linked with the multi-threading support (e.g. pthreads on linux implementing std::mutex). If it works without it, then it must get linked with -lpthreads somehow else.

Reply:
I don't know. You decide what would be better :)